fontforgeexe/fontinfo.h

```diff
@@ -0,0 +1,21 @@
+#ifndef FONTFORGE_FONTFORGEEXE_FONTINFO_H
+#define FONTFORGE_FONTFORGEEXE_FONTINFO_H
+
+#include <ggadget.h>
```

Comment:
It does, and I was wrong about `-I` not being used by default for system headers. The special option is `-iquote`.

I still think using it for non-system headers is poor form. [Elsewhere](https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html) in the gcc documentation, it says that that is for system headers.